execution_engine/src/storage/trie_store/operations/mod.rs

```diff
@@ -256,32 +256,41 @@ where
         if !visited.insert(trie_key) {
             continue;
         }
-        let maybe_retrieved_trie: Option<Trie<K, V>> = store.get(txn, &trie_key)?;
+        let maybe_retrieved_trie_bytes: Option<Bytes> = store.get_raw(txn, &trie_key)?;
```

Comment:
```suggestion
        let trie_value_bytes: Bytes = match store.get_raw(txn, &trie_key) {
           None => { missing_descendants.push(trie_key); continue }
           Some(trie_value_bytes) => trie_value_bytes
        };
```

This simplifies the code downstream...

Reply:
Apparently implemented: https://github.com/dwerner/casper-node/blob/218377c25745e707a9cb939fb0693d3c876c9107/execution_engine/src/storage/trie_store/operations/mod.rs#L259